### PR TITLE
Adding first draft of IntegrationSource doc

### PR DESCRIPTION
**Pull request by matzew**
Work in progress, do not yet merge.

/hold
(for now)

### CODE REVIEW

File: docs/eventing/sources/integration-source/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      ref:
+        apiVersion: eventing.knative.dev/v1
+        kind: Broker
+        name: default
+  ```
+Inside of the `aws.sqs` object we define the name of the queue (or _arn_) and its region. The credentials for the AWS service are referenced from the `my-secret` Kubernetes `Secret` 
+
+More details about the Apache Camel Kamelet [aws-sqs-source](https://camel.apache.org/camel-kamelets/latest/aws-sqs-source.html).
+
+### AWS DynamoDB Streams
```
Comment (pierDipi) [inline docs/eventing/sources/integration-source/README.md:80]: I would split pages for each source, that helps with Google search and general discoverability

Comment (matzew) [reply docs/eventing/sources/integration-source/README.md:80]: OK. thats possible

**Review by Cali0707 — COMMENTED** [review]
Can we link to this in the table of all sources: https://github.com/knative/docs/tree/main/docs/eventing/sources @matzew ?

**Comment by matzew** [comment]
> Can we link to this in the table of all sources: https://github.com/knative/docs/tree/main/docs/eventing/sources @matzew ?

yes, I will add it. but I generally wanted to rework that page. remove triggermesh, and list ours

**Comment by matzew** [comment]
@Cali0707 that page needs a separate PR for cleaning up. there are some inactive sources etc. So I will separate this change.

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

I think we could link to the IntegrationSource API reference in each source page so that we let users know the full spec

**Pull request merged**